ramani-maplibre/src/main/java/org/ramani/compose/Symbol.kt

```diff
@@ -27,11 +26,11 @@ import org.maplibre.android.style.layers.Property.TEXT_JUSTIFY_CENTER
 @MapLibreComposable
 fun Symbol(
     center: LatLng,
-    size: Float,
-    color: String,
-    isDraggable: Boolean,
+    size: Float = 1F,
+    color: String = "",
+    isDraggable: Boolean = false,
     zIndex: Int = 0,
-    imageId: Int? = null,
+    imageId: Int? = org.maplibre.android.R.drawable.maplibre_marker_icon_default,
```

Comment:
It feels like having defaults here makes sense.

Reply:
Agree!